qiskit/aqua/operators/primitive_ops/circuit_op.py

```diff
@@ -139,14 +140,13 @@ def to_matrix(self, massive: bool = False) -> np.ndarray:
                 ' in this case {0}x{0} elements.'
                 ' Set massive=True if you want to proceed.'.format(2 ** self.num_qubits))
 
+        # TODO: Check order of qubits. The note below refers to using the unitary simulator.
+        # But, we have replaced this code with qiskit.quantum_info.Operator
```

Comment:
I assume this is the TODO in question.

And in the Note below `Unitary simulator` becomes `Quantum Info operator` ?

Reply:
Yes and Yes. The question is whether the ordering of qubits differs between the two.